app/uk/gov/hmrc/voapropertylinking/models/modernised/agentrepresentation/AppointmentChangeResponse.scala

```diff
@@ -16,10 +16,16 @@
 
 package uk.gov.hmrc.voapropertylinking.models.modernised.agentrepresentation
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json._
 
 case class AppointmentChangeResponse(appointmentChangeId: String)
 
 object AppointmentChangeResponse {
-  implicit val format: OFormat[AppointmentChangeResponse] = Json.format
+  implicit val reads: Reads[AppointmentChangeResponse] =
```

Comment:
why don't we rename the field like we did on the other end? then there wouldn't be a custom reads/writes and the same thing wouldn't be called different names

Reply:
I did this so that I don't have to rename the frontend as well. This is a fields that we don't really care about, we don't use the value for anything

Reply:
I see your point but im not sure if we should be letting modernised's model bubble all the way up to our frontend. we would be completely coupled

Reply:
in our would its always called "appointmentChangeId". We are just aware that modernised call it "agentAppointmentChangeId"

Reply:
I see... 👌